addons/udes_stock/models/stock_picking_type.py

```diff
@@ -307,6 +307,13 @@ class StockPickingType(models.Model):
              'This is ignored if the number of pickings to reserve is 0.'
     )
 
+    u_auto_unlink_empty = fields.Boolean(
+        string='Auto Unlink Empty',
+        default=True,
+        help='When False, filter out empty pickings of this picking type when '
```

Comment:
Can we explain what the flag does when it's enabled?  The double negative (unlink, False) could be difficult to understand.